### DISCUSSION TITLE
Smart episode a la autodl-irssi

### DISCUSSION THREAD

**Discussion by mkgeeky** [autobrr/autobrr#291] [Ideas]
Another nice feature from autodl-irssi.
Stop downloading same episode again and again.
Eg. if you last downloaded/grapped episode 6, the system will only grap above episode 6 and NOT 6 or below.
Yes, i know about Sonarr and Sonarr works, but i think this chould be nice to have 💪

**Comment by mouzzampk** [comment]
+1 for this and same with movies if one group already downloaded like `EPSiLON` then add others as cross seed or reject anything else afterwards.

**Comment by KyleSanderson** [comment]
We added this in 1.12
